src/index.css

```diff
@@ -0,0 +1,91 @@
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@400;500;600;700&display=swap');
+
+* {
+  color: #ffff;
+  font-family: 'Oswald', sans-serif;
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100vw;
```

Comment:
Here I suggest it is better to use percent like (width: 100%) to avoid any form of a scroll.